index.html

```diff
@@ -0,0 +1,232 @@
+  <title>Wilsan's First Capstone project</title>
+  <link rel="stylesheet" href="style.css">
+
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link
+    href="https://fonts.googleapis.com/css2?family=Inter:wght@400;700&family=Lato:ital,wght@0,300;0,700;1,700&display=swap"
+    rel="stylesheet">
+</head>
+
+<body>
+  <nav>
+    <div class="desktop-nav">
+        <div class="upper-nav">
+            <div class="iconCan d-flex">
+                <a href="#"><img src="Images/facebook.png" alt=""></a>
+                <a href="#"><img src="Images/twitter.svg" alt=""></a>
+
+                <a href="#">English</a>
+                <a href="#">My page</a>
+                <a href="#">Logout</a>
+            </div>
+        </div>
+
+        <div class="main-nav">
+            <div class="subNav d-flex">
+                <div class="subLeft">
+                    <img src="./Images/logo.jpg" alt="">
+                </div>
+                <!-- hamburger -->
+                <div class="ham" id="menu">
+                    <img src="./Images/hamburger.svg" alt="">
+                </div>
+                <!-- moblie nav class moblie abso -->
+                <div class="moblie-abso" id="item">
+                    <!-- close -->
+                    <div class="closes" id="close">
+                        <img src="./Images/close.svg" alt="">
+                    </div>
+                    <div class="subMiddle d-flex">
+                        <a href="about.html" class="active">About</a>
+                        <a href="about.html#prog">Program</a>
+                        <a href="">Join</a>
+                        <a href="">Sponsor</a>
+                        <a href="">News</a>
+                    </div>
+                    <div class="subRight">
+                        <a href="#">
+                            CC campaign
+                        </a>
+                    </div>
+                </div>
+
+            </div>
+        </div>
+    </div>
+
+    <div class="nav-menu">
+        <img src="images/hamburger.svg" class="hamburger" id="menu"
+            alt=" The menu bar for easy access to Portfolio's content">
+        <ul class="menuItems" id="item">
+
+            <div class="listCan" id="list">
+                <li><a href="index.html" class="item" id="first">Home</a></li>
+                <li><a href="#prog" class="item">Program</a></li>
+                <li><a href="" class="item">Join</a></li>
+                <li><a href="" class="item">Sponsor</a></li>
+                <li><a href="" class="item">News</a></li>
+                <li>
+                    <div class="subRight">
+                        <a href="#">
+                            CC campaign
+                        </a>
+                    </div>
+                </li>
+            </div>
+        </ul>
+    </div>
```

Comment:
- [ ] As suggested by the previous code reviewer you need to align your mobile view nav bar to fit the Figma design

Reply:
Please we don't use figma

Reply:
😄 let it for the Behance design
[Cindy Shin in Behance](https://www.behance.net/adagio07)